configurations/eeemcal_only.yml

```diff
@@ -1,7 +1,5 @@
 features:
-  #  beampipe:
-  tracking:
-    definitions_craterlake:
-    support_service_craterlake:
```

Comment:
There are some of these services in front of the EEEMCal, right?

Reply:
I think, the reason I removed the services is because none of the other "_only" configurations include those.

Reply:
Then we have to make sure no one is using them for calorimeter studies.

Reply:
I'm not using them. They were added to get a visual view, and I didn't take them out from the file that was committed. Those services are not in front of EEEMCal.

Reply:
Yes, and the point is to be able to quickly rule-out situations when we can't just blame other detectors :)